src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -224,24 +249,19 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
     val n_c = DenseVector[Double](n_current._1, n_current._2, n_current._3)
     val n_d = DenseVector[Double](n_desired._1, n_desired._2, n_desired._3)
     if (math.abs(linalg.norm(linalg.cross(n_c,n_d))) > NumericUtils.EPSILON) {
-      val (u, v, w) = n_current
+      val v = linalg.cross(n_c, n_d)
+      val s = linalg.norm(v)
+      val c = n_c dot n_d
 
-      // Rotation matrix to rotate into x-z plane
-      val Txz = DenseMatrix.zeros[Double](3, 3)
-      Txz(0,0) = u / math.sqrt(u*u + v*v)
-      Txz(1,0) = -v /math.sqrt(u*u + v*v)
-      Txz(0,1) = v / math.sqrt(u*u + v*v)
-      Txz(1,1) = u / math.sqrt(u*u + v*v)
-      Txz(2,2) = 1.0
+      val v_skew = DenseMatrix.zeros[Double](3,3)
```

Comment:
It's called `v_skew` because it's a skew symmetric matrix. I can change it to something else, but I think at least the skew portion makes sense.